Documentation/markdown/SetupHelp.md

```diff
@@ -86,4 +109,15 @@ Before you can proceed to the Installation of **OpenInfraPlatform**, check, if t
 4. *Build* project **OpenInfraPlatform.UI**.
 5. *Build* all projects staged in the **Copy** project folder. 
 
-	**NOTE:** execute **Get_OKLABI.cmd** in the source folder (e.g. C:\dev\Open-Infra-Platform\external\) to prevent an error during the build process. After the execution of **Get_OKLABI.cmd**, it's necessary to *configure* and *generate* OpenInfraPlatform project once again in CMake.
\ No newline at end of file
+	**NOTE:** execute **Get_OKLABI.cmd** in the source folder (e.g. `C:\dev\Open-Infra-Platform\external`) to prevent an error during the build process. After the execution of **Get_OKLABI.cmd**, it's necessary to *configure* and *generate* OpenInfraPlatform project once again in CMake.
+
+
+## <a name="Generating_Doc"></a> Generating documentation
+
+### <a name="Get_Dot"></a> Get Dot and Doxygen
+
+1. After cloning **OpenInfraPlatform** repository open OpenInfraPlatform source folder. 
+2. Open folder **external**
+3. Open files:
+	* GetDot.cmd  
+	* GetDoxygen.cmd
```

Comment:
Add a reference/link to Doxygen help document after this line, sth like "Read more about Doxygen in our guidelines".

Reply:
Also, always end any file within our repository with an empty line (that is, the last line needs to be empty).

Reply:
Resolved:  a4a7d9c